.changeset/all-canyons-cross.md

```
@@ -0,0 +1,5 @@
+---
+"@idos-network/kwil-infra": minor
+---
+
+Allow verify ETH signature offline first, then as a fallback use wagmi and eth_call to validate Safe wallets
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)